Sentry.xcodeproj/project.pbxproj

```diff
@@ -4558,6 +4566,8 @@
 				CLANG_ANALYZER_NONNULL = YES;
 				CLANG_ANALYZER_NUMBER_OBJECT_CONVERSION = YES;
 				CLANG_ENABLE_OBJC_ARC = YES;
+				CLANG_ENABLE_OBJC_WEAK = YES;
+				CLANG_MODULES_AUTOLINK = NO;
```

Comment:
This is one of the crucial parts. It requires explicitly linking CoreData and SystemConfiguration frameworks.